openmls/src/framing/validation.rs

```diff
@@ -40,6 +40,9 @@ use crate::ciphersuite::signable::Verifiable;
 
 use super::*;
 
+/// Contains a [VerifiableMlsPlaintext]. Can be built either from a plaintext or ciphertext.
+/// In the latter case, it attempts to decrypt the ciphertext.
+/// Checks the presence of the membership tag and confirmation tag.
```

Comment:
Some of this feels like it should be documentation for the functions that are part of this struct's implementation. For example, `... it attempts to decrypt the ciphertext`. What is it? Same for the last sentence.